src/applications/disability-calculator/components/RatingRow.jsx

```diff
@@ -50,18 +49,17 @@ const RatingRow = ({
         />
       </div>
       <div className="vads-l-col--3">
-        {canDelete && (
-          <div>
-            <button
-              type="button"
-              onClick={handleRemoveRating(indx)}
-              className="va-button-link delete-btn vads-u-margin--1p5"
-            >
-              <i className="fas fa-trash-alt vads-u-padding-right--0p5" />
-              Delete
-            </button>
-          </div>
-        )}
+        <div>
+          <button
+            type="button"
+            onClick={handleRemoveRating(indx)}
+            className="va-button-link delete-btn vads-u-margin--1p5"
+            disabled={canDelete < 3}
+          >
+            <i className="fas fa-trash-alt vads-u-padding-right--0p5" />
+            Delete
+          </button>
+        </div>
```

Comment:
A couple things - 

1. I don't think we need `canDelete` anymore. I think we can just check the index of this row in the array, which you are already passing via props as `idx`. 

2. Instead of disabling it, maybe we should just hide the button altogether if it's not deletable. We can do that with a bit of logic inside the JSX there, like -

```jsx
{this.props.idx > 2 && (
<button
    type="button"
    onClick={handleRemoveRating(indx)}
    className="va-button-link delete-btn vads-u-margin--1p5"
    >
        <i className="fas fa-trash-alt vads-u-padding-right--0p5" /> Delete
    </button>
)}
```

Reply:
We hid the buttons initially, but after stakeholder review, they wanted the delete buttons on the first two rows.